java/ql/src/semmle/code/java/security/FileReadWrite.qll

```diff
@@ -25,6 +25,8 @@ private predicate fileRead(VarAccess fileAccess, Expr fileReadingExpr) {
       (
         filesMethod.hasName("readAllBytes") or
         filesMethod.hasName("readAllLines") or
+        filesMethod.hasName("readString") or
```

Comment:
I think you could refactor that to use [set-literals](https://help.semmle.com/QL/ql-handbook/expressions.html#set-literal-expressions) instead.
So:
```ql
filesMethod.hasName(["readAllBytes", "readAllLines", "readString",
"lines", "newBufferedReader", "newInputReader", "newByteChannel"])
```

Reply:
Thanks, I wasn't aware that you could use sets like this and hadn't considered changing the structure of the predicate yet, but it makes sense here indeed.

Maybe having each name in a separate line would allow better readability. Though I am not sure which format would be allowed by the style guide.

Also feel free to create a GitHub suggestion comment here on this pull request for these lines performing your suggested change and will add it.

Reply:
I can't.
Only maintainers of this repository can add these suggestions.

I'm not a maintainer (or affiliated with lgtm/semmle/github/microsoft) although the -lgtm part of my username my suggest that^^
I created this `intrigus-lgtm` account for various reason when I first used lgtm.com and since have been using it for anything security related.

Reply:
You should be able to write suggestion comments, see e.g. [my suggestion](https://github.com/github/codeql/pull/3794#discussion_r446281722) on your pull request. And I assume there is also a "Commit" button visible for you.